pyproject.toml

```diff
@@ -22,7 +22,7 @@ addopts = "-n auto --cov=. --cov-config=../pyproject.toml -p no:unraisableexcept
 filterwarnings = [
     "error",
     "ignore:(?s).*Implementing implicit namespace package:DeprecationWarning:pkg_resources",
-    "ignore:pkg_resources is deprecated as an API:DeprecationWarning:pkg_resources",
+    "ignore:pkg_resources is deprecated as an API.:DeprecationWarning:pkg_resources",
```

Comment:
Following Thibault comment, I think we would have to change (with another unkown vlaue for me at this point) or remove the `pkg_resources` to make it work

Reply:
I can't get this warning locally.. Perhaps because I'm on 3.10? 🤔

Reply:
Nope, that wasn't it.